e2e/specs/amend-listing-request-with-notes.e2e-spec.ts

```diff
@@ -0,0 +1,95 @@
+
+const loginFlow: LoginFlow = new LoginFlow();
+const navigationFlow: NavigationFlow = new NavigationFlow();
+const sessionSearchPage = new SessionSearchPage();
+const listingCreationPage = new ListingCreationPage();
+const transactionDialogPage = new TransactionDialogPage()
+
+const caseNumber = `number-${new Date().toLocaleString()}`
+const caseTitle = `title-${new Date().toLocaleString()}`
+const caseType: string = CaseTypes.SCLAIMS;
+const otherCaseType = CaseTypes.FTRACK;
+const duration = 'PT30M';
+const otherDuration = 45;
+const priority = 'Low';
+const id = uuid();
+const userTransactionId = uuid();
+const hearingType = HearingTypes.TRIAL;
+const otherHearingType = HearingTypes.ADJOURNED;
+const specReqNoteValue = 'SPEC REQ';
+
+const displayedListingRequestData = {
+  caseNumber,
+  caseTitle,
+  caseType,
+  hearingType,
+  priority
+};
+
+const listingRequestCreate: CreateListingRequestBody = {
+    id, caseNumber, caseTitle, priority, duration, userTransactionId, caseTypeCode: 'small-claims',  hearingTypeCode: 'trial'
+}
+
+describe('Amend Listing Request with notes', () => {
+  beforeAll(async () => {
+    await loginFlow.relogin()
+  });
+  describe('Create Listing Request via API', () => {
+    it('should create listing request', async () => {
+      const statusCode = await API.createListingRequest(listingRequestCreate)
+      expect(statusCode).toEqual(200);
+    });
+  });
+  describe('Go to listing hearings page', () => {
+    it('created listing request should be visible on list', async () => {
+      await navigationFlow.goToListHearingsPage();
+      await sessionSearchPage.changeMaxItemsPerPage('100');
+      const isListingRequestDisplayed = await sessionSearchPage.isListingRequestDisplayed(...Object.values(displayedListingRequestData));
+      expect(isListingRequestDisplayed).toBeTruthy();
+    });
+  });
+  describe('Click on edit and change some values and a note', () => {
+    it('new values should be visible on list', async () => {
+      await sessionSearchPage.editListingRequestWithValues(caseNumber);
+      const newCaseNumber = `edited-${caseNumber}`;
+      const newCaseTitle = `edited-${caseTitle}`;
+      const listingForm: ListingCreationForm = {
+        caseNumber: newCaseNumber,
+        caseTitle: newCaseTitle,
+        caseType: otherCaseType,
+        hearingType: otherHearingType,
+        duration: otherDuration,
+        fromDate: todayDate,
+        endDate: tomorrowDate
+      };
+      await listingCreationPage.setNoteValue(LISTING_NOTES.SPECIAL_REQUIREMENTS, specReqNoteValue);
+      await listingCreationPage.createListingRequest(listingForm);
+      await transactionDialogPage.clickAcceptButton();
+      const isListingRequestDisplayed = await sessionSearchPage.isListingRequestDisplayed(...Object.values(listingForm));
+      expect(isListingRequestDisplayed).toBeTruthy()
+    });
+  });
+
+    describe('Click on notes indicator and verify the notes', () => {
```

Comment:
It 80% copy of amending listing request e2e. Can you extend the test that we already have? Or is there a reason for having a separate one?

Reply:
Agreed with @arompa to create a separate one. It does duplicate lots of previous test. Maybe it's worth revisiting or rediscussing with Ania